agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -436,8 +443,12 @@ func (cs *CredentialSpecResource) handleSSMCredentialspecFile(originalCredential
 
 	ssmParamData := ssmParamMap[ssmParam]
 
-	localCredSpecFilePath := fmt.Sprintf("%s/ssm_%s_%s.json", cs.credentialSpecResourceLocation, cs.taskARN, ssmParam)
-
+	taskArnSplit := strings.SplitN(cs.taskARN, "/", 2)
+	if (len(taskArnSplit) != 2) {
+		seelog.Errorf("Failed to retrieve taskId from taskArn.")
```

Comment:
Can you add seelog.Errorf() to the code places that calling **handleSSMCredentialspecFile/handleS3CredentialspecFile/handleCredentialspecFile**, which is in the create function, instead of just adding a single error log printing here.

Reply:
Done

Reply:
nit: Why are you logging the same message twice?